client/src/pages/Messages/Chat/Chat.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { Avatar, Box, Button, FilledInput, FormControl, Typography } from '@mui/material';
+import moment from 'moment';
+import { Badge } from '@mui/material';
+import createNewMessage from '../../../helpers/APICalls/createNewMessage';
+import useStyles from './useStyles';
+import SendIcon from '@mui/icons-material/Send';
+
+const Chat = (conversation: any) => {
+  const classes = useStyles();
+  const [text, setText] = useState('');
+  const handleChange = (e: any) => {
+    setText(e.target.value);
+  };
+  const handleSubmit = () => {
+    const data = {
+      text,
+      conversationId: conversation.conversationId as string,
+    };
+    createNewMessage(data).then((data) => {
+      if (data.error) {
+        console.error({ error: data.error.message });
+      } else if (data.success) {
+        const conversationData = {
+          conversationId: conversation.conversationId,
+        };
+        conversation.setNewMessage(true);
+        conversation.fillMessageDetails(conversation.conversationId);
+        setText('');
+      }
+    });
+  };
+  return (
+    <Box className={classes.root}>
+      <>
+        <Box className={classes.root1}>
+          <Box className={classes.content}>
+            <Badge anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}>
+              <Avatar src={conversation.message.photo} className={classes.profilePic}></Avatar>
+            </Badge>
+            <Typography
+              sx={{
+                fontWeight: 700,
+                marginLeft: 1,
+              }}
+              className={classes.username}
+            >
+              {conversation.message.name}
+            </Typography>
+          </Box>
+        </Box>
+        <Box className={classes.chatContainer}>
+          <Box>
+            {conversation.messages.map((message: any) => {
+              const time = moment('12/12/32').format('h:mm');
+
+              return conversation.profileId === message.senderId ? (
+                <Box className={classes.root2}>
```

Comment:
It would make the code more maintainable if we could componentize the chat bubble